Defines the most popular GENTOO_MIRRORS setting throughout the
administrative realm, enabling multiple sites to use them
as a shared patch.

Index: etc/make.conf
===================================================================
--- etc.orig/make.conf
+++ etc/make.conf
@@ -524,4 +524,4 @@ SSL_UNIT="Preliminary usage"
 # This entry should be the last in /etc/make.conf!
 #
 # Run "mirrorselect -s10 -b10" in order to update the next line.
-#GENTOO_MIRRORS=""
+GENTOO_MIRRORS="/usr/local/portage/distfiles/local /usr/local/portage/distfiles /usr/local/portage/distfiles/precious /usr/local/portage/distfiles/mnt http://mirror.its.uidaho.edu/pub/gentoo/ ftp://ftp.swin.edu.au/gentoo ftp://mirror.its.uidaho.edu/gentoo/ http://ftp.swin.edu.au/gentoo http://cesium.di.uminho.pt/pub/gentoo/ ftp://mirror.pacific.net.au/linux/Gentoo ftp://cesium.di.uminho.pt/pub/gentoo/ http://gentoo.wetzlmayr.com/ http://mirror.pacific.net.au/linux/Gentoo http://gentoo.in.th/"
